fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -3011,6 +3058,7 @@ struct TagPartitionedLogSystem : ILogSystem, ReferenceCounted<TagPartitionedLogS
 		for (int i = 0; i < logSystem->tLogs[0]->logServers.size(); i++)
 			recoveryComplete.push_back(transformErrors(
 			    throwErrorOr(
+					// question: below here, there are usages of `logServers` and its unique field, e.g. recoveryFinished, what should we do about new interface?
```

Comment:
Please add TLogRecoveryFinishedRequest serving into ptxn TLogServer and here we need to send TLogRecoveryFinishedRequest to new tlog servers if the knob is true.

Reply:
will discuss offline